tools/api_proto_breaking_change_detector/detector_test.py

```diff
@@ -25,8 +40,14 @@ def run_detector_test(self, testname, is_breaking, expects_changes, additional_a
         current = Path(tests_path, f"{testname}_current")
         changed = Path(tests_path, f"{testname}_next")
 
-        detector_obj = self.detector_type(current, changed, additional_args)
-        detector_obj.run_detector()
+        # buf requires protobuf files to be in a subdirectory of the yaml file
+        with tempfile.TemporaryDirectory(prefix=str(Path.cwd()) + os.sep) as temp_dir:
```

Comment:
This is weird, we're appending a `/` here? Is that necessary?

Reply:
ah I thought `prefix` was the only way to do this (in which case yeah it would be necessary) but I just found out there is a `dir` argument which is really what I want here